pkg/features/features_test.go

```diff
@@ -41,5 +41,8 @@ var _ = Describe("Init Action", func() {
 			_, err := features.Get([]string{"unknown-abc"})
 			Expect(err).To(HaveOccurred())
 		})
+		It("Installs a feature correctly", func() {
+
```

Comment:
Test missing here?

Reply:
Hmm, this code is covered by `pkg/action/init_test.go`, removing this!